main.tf

```diff
@@ -1909,7 +1909,11 @@ module "cert_manager" {
   postrender = try(var.cert_manager.postrender, [])
   set = concat([
     {
-      name  = "installCRDs"
+      name  = "crds.enabled"
```

Comment:
I don't think this change on its own is valid - I suspect this is tied to a new major version which would be a breaking change